tests/GingerTest.php

```diff
@@ -31,8 +31,28 @@ public function itShouldCreateAClient()
             $httpClient->getDefaultOption('headers')
         );
         $this->assertEquals(
-            ['my-api-key', ''],
+            ['f47ac10b58cc4372a5670e02b2c3d479', ''],
             $httpClient->getDefaultOption('auth')
         );
     }
+
+    /**
+     * @test
+     */
+    public function itShouldFailWithIncorrectAPIkey()
+    {
+        $this->setExpectedException('Assert\InvalidArgumentException');
+        Ginger::createClient('my-api-key');
+    }
+
+    /**
+     * @test
+     */
+    public function itShouldCreateAValidUUD()
```

Comment:
Typo: UUD -> UUID

Also, shouldn't it be `Uuid` instead, to be consistent with `apiKeyToUuid `?